packages/search-ui-app-search-connector/src/buildResponseAdapterOptionsFromConfig.js

```diff
@@ -0,0 +1,22 @@
+export default function buildResponseAdapterOptionsFromState(config) {
+  const additionalFacetValueFields = Object.entries(config.facets || {}).reduce(
```

Comment:
Do we need to check for whether `config` is an obj (so that `config.facets` doesn't throw a type error) or is that probably overkill?